roles/provision-data-sync-apb/tasks/provision-sync-server.yml

```diff
@@ -12,6 +12,8 @@
     spec_template_metadata_labels:
       app: data-sync
       service: data-sync-server
+    spec_template_metadata_annotations:
+      org.aerogear.metrics/plain_endpoint: /metrics
     containers:
     - name: data-sync-server
       image: '{{ sync_server_image }}:{{ sync_server_version }}'
```

Comment:
Container port needs to be specified, otherwise prometheus is not able to connect and get data from `/metrics` endpoint
```
          ports:
           - containerPort: 8000
             protocol: TCP
```

Reply:
having a look again.